tests/unit/src/lib/validator.test.js

```diff
@@ -0,0 +1,116 @@
+const { isValidNino } = require("../../../../src/lib/validator");
+
+const good_ninos = [
+  "AA283902A",
```

Comment:
Question for @merlinc : Since this is a public repo and if by chance these are real NiNo's, would this be a problem?

Reply:
Interesting question. We're using this list as known valid, but really the value is making sure that known incorrect ninos are detected. 

Maybe we can drop this list down to only a few entries?

Reply:
Yeah, I think dropping the number to a more concise list might be better.